### PR TITLE
CircleCI integration

### PR DESCRIPTION
**Pull request by jonmanahan**
### Summary:
- Added CircleCI config file
- Tested that it would both pass and fail given passing and failing tests respectively

### CODE REVIEW

File: test/tic_tac_toe_test.exs [hunk 1]
```diff
@@ -4,4 +4,9 @@ defmodule TicTacToeTest do
   test "passes the test automatically" do
     assert true
   end
+
+  @tag :skip
+  test "fails the test automatically" do
+    assert false
+  end
```
Comment (elijahsullivan) [inline test/tic_tac_toe_test.exs:11]: You can remove this now if you want. I can look through the history of CircleCI jobs to see that the failing test failed the workflow https://app.circleci.com/pipelines/github/jonmanahan/Tic-Tac-Toe?branch=circle_ci&filter=all

Comment (jonmanahan) [reply test/tic_tac_toe_test.exs:11]: Sounds good to me, Ill remove that and push it to this branch! Would that be ready to merge after?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: "cimg/elixir:1.13.3"
+    steps:
+      - checkout
+      - run: elixir ./lib/tic_tac_toe.ex
```
Comment (elijahsullivan) [inline .circleci/config.yml:9]: Could you replace this with a `mix` command that achieves the same objective? What if the entry point of the application were to change from `tic_tac_toe.ex`?

Comment (jonmanahan) [reply .circleci/config.yml:9]: That is a good point that I didn't think of, will do!

**Review by elijahsullivan — APPROVED** [review]

**Pull request merged**